src/stream/src/from_proto/source.rs

```diff
@@ -79,6 +79,8 @@ impl ExecutorBuilder for SourceExecutorBuilder {
             params.op_info,
             params.executor_stats,
             stream_source_splits,
+            // TODO: move checkpoint interval to stream config
```

Comment:
Should decide according to the barrier interval on meta?

Reply:
Yes, I'll load it from config in the future.